public/index.html

```diff
@@ -53,5 +76,8 @@
         <div class="mt-4 mx-3 category">Accessories</div></a
       >
     </div>
+    <script defer src="/js/common.js"></script>
+    <script defer src="js/home.js"></script>
+    <script src="./js/nav.js"></script>
```

Comment:
why did we move these to the bottom?

Reply:
Before running the script HTML elements should be loaded which can prevent some error like 'undefined' due to executing the script before the element is loaded.